javalib/src/main/scala/java/net/AbstractPlainDatagramSocketImpl.scala

```diff
@@ -0,0 +1,786 @@
+package java.net
+
+import java.net.ipOps._
+import java.io.{FileDescriptor, IOException}
+import scala.scalanative.libc.string.memcpy
+import scala.scalanative.meta.LinktimeInfo.{isLinux, isWindows}
+import scala.scalanative.posix
+import scala.scalanative.posix.arpa.inet
+import scala.scalanative.posix.errno._
+import scala.scalanative.posix.netdb._
+import scala.scalanative.posix.netdbOps._
+import scala.scalanative.posix.netinet.in
+import scala.scalanative.posix.netinet.inOps._
+import scala.scalanative.posix.sys.ioctl.{FIONREAD, ioctl}
+import scala.scalanative.posix.sys.socketOps._
+import scala.scalanative.posix.sys.time._
+import scala.scalanative.posix.sys.timeOps._
+import scala.scalanative.posix.unistd
+import scala.scalanative.unsafe._
+import scala.scalanative.unsigned._
+import scala.scalanative.windows.WinSocketApi.WSAGetLastError
+import scala.scalanative.windows._
+import scala.scalanative.windows.WinSocketApi._
+import scala.scalanative.windows.WinSocketApiExt._
+
+private[net] abstract class AbstractPlainDatagramSocketImpl
+    extends DatagramSocketImpl {
+  import AbstractPlainDatagramSocketImpl._
+
+  protected def tryPoll(op: String): Unit
+
+  protected[net] var fd = new FileDescriptor
+  protected[net] var localport = 0
+  protected[net] var address: InetAddress = null
```

Comment:
Whenever possible use `_` initialiser for initially null vars. `= null` can sometimes backfire when modifying the code. If lhs would be previously assigned `= null` would erase it. On the other hand `= _` would skip already assigned field.

Reply:
Thanks. TIL something new!